### PR TITLE
Miscellaneous fixes: make clean fix, adding argument passing to tests, fixing typo in configure-mana

### PR DESCRIPTION
**Pull request by chirag-singh-memverge**

### CODE REVIEW

File: mpi-proxy-split/test/mana_test.py [hunk 1]
```diff
@@ -43,16 +45,26 @@ def main():
...
+                                 f'{args.test}.mana.exe'
+                                 f'{args.args}'])
     else:
-        print(f'{run} -n {args.num_ranks} {mana_launch_path} '
-               f'{args.test}.mana.exe {args.iterations}')
-        test_child = subprocess.run([f'{run}', '-n', f'{args.num_ranks}',
-                                 f'{mana_launch_path}',
-                                 f'{args.test}.mana.exe', f'{args.iterations}'])
+        if args.args == '':
+            print(f'{run} -n {args.num_ranks} {mana_launch_path} '
```
Comment (JainTwinkle) [inline mpi-proxy-split/test/mana_test.py:55]: print can come out of the if...else!

Comment (chirag-singh-memverge) [reply mpi-proxy-split/test/mana_test.py:55]: It's different for each note the -i

Comment (JainTwinkle) [reply mpi-proxy-split/test/mana_test.py:55]: I see it now.

File: mpi-proxy-split/Makefile [hunk 2]
```diff
@@ -128,7 +128,9 @@ tidy:
 clean: tidy
 	rm -f ${LIBOBJS} ${MANA_COORD_OBJS}
 	rm -f libmana.so
-	rm -f ${MANA_ROOT}/lib/dmtcp/libmana.so
+	if test -d ${MANA_ROOT}/lib/dmtcp/libmana.so; then \
```
Comment (JainTwinkle) [inline mpi-proxy-split/Makefile:131]: `-d` is for directory and `libmana.so` is a file, not directory.

Comment (JainTwinkle) [reply mpi-proxy-split/Makefile:131]: and rm -f  works whether you the file exists or not. I don't think you need to test anything here.

**Comment by JainTwinkle** [comment]
While you are at it, could you please correct https://github.com/mpickpt/mana/blob/447a78a109fca001fe9193fe9fab62379b04a81f/configure-mana#L13?
There's an extra underscore in the variable name?

**Comment by JainTwinkle** [comment]
Let's change the PR title and description to Misc. changes.

**Pull request merged**